README.md

```diff
@@ -48,18 +48,18 @@ $ kubectl apply -f deploy/upstream
 ```
 
 #### Installing an operator using Marketplace
-The following section assumes that Marketplace was installed in the `openshift-marketplace` namespace. For Marketplace to function you need to have at least one `OperatorSource` CR present on the cluster. To get started you can use the [community](deploy/examples/operatorsource.cr.yaml) OperatorSource. If you are on an OKD cluster, you can skip this step as it is installed by default.
+The following section assumes that Marketplace was installed in the `openshift-marketplace` namespace. For Marketplace to function you need to have at least one `OperatorSource` CR present on the cluster. To get started you can use the [upstream-community-operators](deploy/examples/operatorsource.cr.yaml) OperatorSource. If you are on an OKD cluster, you can skip this step as it is installed by default.
```

Comment:
Given you are targeting this for upstream, please update the namespace to `marketplace`. It is better to say `use the OperatorSource` for upstream community operators. Please use `` when referring to `OperatorSource`.

Also update what is installed by default on an OKD cluster.